packages/squiggle-lang/src/library/registry/frTypes.ts

```diff
@@ -467,24 +464,33 @@ export function frDict<T extends object>(
   };
 }
 
-// Optionals are implemented for the sake of frDict, which check for them explicitly.
-// Don't try to use them in other contexts.
-export const frOptional = <T>(
-  itemType: FRType<T>,
-  paramName?: string
-): FRType<T | null> & { isOptional: boolean } => {
+export const frNamed = <T>(name: string, itemType: FRType<T>): FRType<T> => ({
+  unpack: itemType.unpack,
+  pack: (v) => {
+    if (v === null) {
+      throw new Error("Unable to pack null value");
+    }
```

Comment:
This seems to be copy-pasted from `frOptional`? Can be replaced with a simple `pack: itemType.pack`.